backend/src/core/models/game.spec.js

```diff
@@ -228,19 +229,67 @@ describe("Game", function () {
       });
     });
 
-    describe("if the provided data does not include a proper release date", function () {
-      beforeAll(function () {
-        this.result = Game.fromSteamApi(theLastNightSteamApiData);
+    describe("if the provided steam api app includes a release date", function () {
+      describe("and the date is a valid date", function () {
+        beforeAll(function () {
+          this.result = Game.fromSteamApi(eldenRingSteamApiData);
+        });
+
+        it("the game's release date will have the correct value", function () {
+          expect(this.result.releaseDate).toEqual(new Date("24 February 2022 UTC"));
+        });
       });
 
-      it("the result is an instance of game", function () {
-        expect(this.result).toBeInstanceOf(Game);
+      describe("and the date is not a valid date", function () {
+        beforeAll(function () {
+          this.result = Game.fromSteamApi(theLastNightSteamApiData);
+        });
+
+        it("the game's release date will be an empty string", function () {
+          expect(this.result.releaseDate).toBe("");
```

Comment:
Better `undefined` or `null`. Empty string should be the default/null value of strings only. And because Date is a class its default/null value is `null`. If the change is to big, create an issue.

Reply:
Created issue for this: #228